histomicsui/web_client/templates/panels/annotationSelector.pug

```diff
@@ -76,6 +76,7 @@ block content
       input#h-toggle-interactive(type='checkbox', checked=interactiveMode)
       | Interactive
     if accessLevel >= writeAccessLevel
-      button.btn.btn-sm.btn-primary.h-create-annotation
-        | #[span.icon-plus-squared] New
+      label(title='Create a new annotation. Keyboard shortcut: space bar')
```

Comment:
The `label` element adds a bunch of css that we probably don't want.  I think changing it to a `span` or just adding the `title` to the button would be better,